x/gov/simulation/operations_test.go

```diff
@@ -91,18 +117,21 @@ func TestWeightedOperations(t *testing.T) {
 		opMsgRoute string
 		opMsgName  string
 	}{
-		{0, types.ModuleName, simulation.TypeMsgSubmitProposal},
-		{1, types.ModuleName, simulation.TypeMsgSubmitProposal},
-		{2, types.ModuleName, simulation.TypeMsgSubmitProposal},
 		{simulation.DefaultWeightMsgDeposit, types.ModuleName, simulation.TypeMsgDeposit},
 		{simulation.DefaultWeightMsgVote, types.ModuleName, simulation.TypeMsgVote},
 		{simulation.DefaultWeightMsgVoteWeighted, types.ModuleName, simulation.TypeMsgVoteWeighted},
 		{simulation.DefaultWeightMsgCancelProposal, types.ModuleName, simulation.TypeMsgCancelProposal},
+		{0, types.ModuleName, simulation.TypeMsgSubmitProposal},
+		{1, types.ModuleName, simulation.TypeMsgSubmitProposal},
+		{2, types.ModuleName, simulation.TypeMsgSubmitProposal},
+		{0, types.ModuleName, simulation.TypeMsgSubmitProposal},
+		{1, types.ModuleName, simulation.TypeMsgSubmitProposal},
+		{2, types.ModuleName, simulation.TypeMsgSubmitProposal},
 	}
 
 	for i, w := range weightesOps {
-		operationMsg, _, _ := w.Op()(r, app.BaseApp, ctx, accs, ctx.ChainID())
-		// require.NoError(t, err) // TODO check if it should be NoError
+		operationMsg, _, err := w.Op()(r, app.BaseApp, ctx, accs, ctx.ChainID())
+		require.NoError(t, err)
```

Comment:
Checking the error breaks with gas error: `out of gas in location: block gas meter; gasWanted: 10000000, gasUsed: 799775: out of gas`.
I do think we do need to check it, so checking this more.